File: HackerEarth/CubeChange.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main()
{
	int test_cases, counter;
	unsigned long long int sides;
	scanf("%d", &test_cases);
	for(counter=0;counter<test_cases;counter++)
	{
		scanf("%llu", &sides);
		if (sides==1)
		{
			printf("%d\n", 1);
		}
		else
		{
			printf("%llu\n", 6*sides*sides-12*sides+8);
		}
	}
	return 0;
}

File: HackerEarth/CubeChange.java
import java.math.*;
import java.util.Scanner;

public class CubeChange
{
    public static void main(String args[])
    {
	int test_cases, counter, sides_temp;
	BigInteger sides;
        Scanner scan = new Scanner(System.in);
	test_cases = scan.nextInt();
	for(counter=0;counter<test_cases;counter++)
	{
		sides_temp = scan.nextInt();
		sides = new BigInteger(Integer.toString(sides_temp));
		if(sides_temp == 1)
		{
			System.out.println("1");
		}
		else
		{
			System.out.println((((sides.pow(2)).multiply(BigInteger.valueOf(6))).add(sides.multiply(BigInteger.valueOf(-12)))).add(BigInteger.valueOf(8)));
		}
	}
    }
}
